src/java.base/share/classes/javax/crypto/KEM.java

```diff
@@ -708,7 +708,7 @@ public Encapsulator newEncapsulator(PublicKey publicKey, SecureRandom secureRand
      * If any extra information inside this object needs to be transmitted along
      * with the key encapsulation message so that the receiver is able to create
      * a matching decapsulator, it will be included as a byte array in the
-     * {@link Encapsulated#params} field inside the encapsulation output.
+     * {@link Encapsulated#params()} field inside the encapsulation output.
```

Comment:
Good catch. The text must have been added before we changed `Encapsulated` from a record to a normal class. Now that `params` is a method instead of a record field, please consider rewriting the sentence to something like
```
it will be included as a byte array returned by the {@link Encapsulated.params()} method within the encapsulation output.
```